skimage/exposure/hdr.py

```diff
@@ -0,0 +1,308 @@
+import numpy as np
+
+
+def get_radiance(images, exposure, radiance_map):
+    """
+    Return the radiance for a series of images based upon a camera response
+    function.
+
+    Parameters
+    ----------
+    images: list
+        List of images in the for of numpy arrays. Either mono or color
+        (RGB, MxNx3).
+    exposure : numpy 1D array
+        Array of exposure times in seconds.
+    radiacenMap : numpy array
+        Array mapping the counts to radiance
+
+    Returns
+    -------
+    hdr : numpy array
+        Resulting image with radiance values.
+    """
+
+    den = np.ones(images[0].shape)
+    num = np.zeros(images[0].shape)
+    wf = np.vectorized(_weight_func)
+    for idx, im in enumerate(images):
+        gij = im.copy()
+        # For colour
+        if im.ndim == 3:
+            for ii in range(im.shape[2]):
+                gij[:, :, ii] = radiance_map[im[:, :, ii] + 1, ii]
+        else:
+            gij[:, :] = radiance_map[im[:, :, ii] + 1]
+        gij = gij - np.log(exposure[idx])
+
+        wij = wf(gij)
+        num = num + wij * gij
+        den = den + wij
+
+    return num / den
+
+
+def make_hdr(images, exposure, radiance_map, depth=16):
+    """
+    Compute the HDR image from a series of images and a racianceMap.
```

Comment:
Add a description in the docstring of how to obtain exposures.  E.g., show how to load exif information from jpegs.